src/templates/auth_manifests.yaml.j2

```diff
@@ -1,15 +1,73 @@
-kind: ClusterRole
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  labels:
+    kustomize.component: profiles
+  name: profiles-controller-service-account
+  namespace: {{ model_name }}
```

Comment:
I think this needs to be replaced with whatever `ServiceAccount` is associated with the workload we deploy for kubeflow profiles.  In this case where it is a sidecar charm, I think we'd want to use the juju-provided `ServiceAccount` (which has the same name as the deployed app)

Reply:
The we will just not apply this `ServiceAccount` and we should be good as long as we have trust.

Reply:
Resolved.